libraries/fiff/fiff_dig_point_set.cpp

```diff
@@ -299,3 +299,24 @@ FiffDigPointSet &FiffDigPointSet::operator<<(const FiffDigPoint *dig)
     this->m_qListDigPoint.append(*dig);
     return *this;
 }
+
+//*************************************************************************************************************
+
+void FiffDigPointSet::applyTransform(const FiffCoordTrans& coordTrans, bool bApplyInverse)
```

Comment:
Which kind of transform is this? Didn't we had this before?

Reply:
No, this functionality was not present before.